### ISSUE TITLE
WALLET: Offer ability to send all funds (taking fee into account)

### ISSUE THREAD

**Issue by DavidBurkett** [GrinPlusPlus/GrinPlusPlus#77]
Currently, it appears it's not even possible to send the last 0.000000001 grin. Add a feature to send all of your funds (minus the fee).

**Comment by hogwashpz** [comment]
I just noticed this, was trying to send all GRINS but even when i substracted 0.000000001 GRIN it still was insufficient funds.

Real example
UTXO:
3.017639417
3.044273884

Fee at the moment:
0.007

Trying to send:
6.061913301 - 0.007 = 6.054913301‬
even 6.054913300 still was insufficient.

Tried restarting the wallet but still fee 0.007 and it still wont send.
Then I send it all with core Grin-wallet, fee was 0.004 and it all gone to new wallet (over http to wallet listening in VM).
